JamGame/Assets/Scripts/Level/DailyBill/Model.cs

```diff
@@ -0,0 +1,35 @@
+﻿using Common;
+using System.Linq;
+using TileBuilder;
+using UnityEngine;
+using UnityEngine.Events;
+
+namespace Level.DailyBill
+{
+    [SerializeField]
+    public struct Check
+    {
+        public int Rent;
+        public int Sum => Rent;
+    }
+
+    [AddComponentMenu("Scripts/Level.DailyBill.Model")]
+    public class Model : MonoBehaviour
+    {
+        [SerializeField]
+        private TileBuilderImpl tileBuilder;
+
+        [SerializeField, InspectorReadOnly]
+        private Check check = new();
+        public Check Check => check;
```

Comment:
We don't need it, as it may be invalid so just call `ComputeCheck()` everywhere when it's needed (it will almost not hit performance)